lib/pbio/drv/uart/uart_stm32f0.c

```diff
@@ -178,6 +178,19 @@ pbio_error_t pbdrv_uart_set_baud_rate(pbdrv_uart_dev_t *uart_dev, uint32_t baud)
     return PBIO_SUCCESS;
 }
 
+void pbdrv_uart_flush(pbdrv_uart_dev_t *uart_dev) {
```

Comment:
Since this also cancels read and write, I would be inclined to call it "reset" rather than "flush". When talking about input flush usually means "discard the buffered data" but for output it usually means "make sure it gets sent right now".

Or maybe the canceling doesn't really belong here?

Reply:
Ack, will rename to reset. I wasn't sure about the read/write cancel -- that was one reason for requesting the review.

Reply:
If it works without the cancel in here, then I think I would prefer to leave them out and leave this as a flush function.

Reply:
It seems we can still get UART/UART to work right without them, which is good. I'm not sure about the passive case, but that should be unrelated, so I'll probably address that as part of the other comments.